app/src/main/kotlin/com/wire/android/ui/common/groupname/GroupConversationNameComponent.kt

```diff
@@ -0,0 +1,140 @@
+package com.wire.android.ui.common.groupname
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ChevronRight
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.wire.android.BuildConfig
+import com.wire.android.R
+import com.wire.android.ui.common.Icon
+import com.wire.android.ui.common.ShakeAnimation
+import com.wire.android.ui.common.WireDropDown
+import com.wire.android.ui.common.button.WireButtonState
+import com.wire.android.ui.common.button.WirePrimaryButton
+import com.wire.android.ui.common.groupname.GroupNameMode.CREATION
+import com.wire.android.ui.common.textfield.WireTextField
+import com.wire.android.ui.common.textfield.WireTextFieldState
+import com.wire.android.ui.common.topappbar.WireCenterAlignedTopAppBar
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.theme.wireTypography
+import com.wire.kalium.logic.data.conversation.ConversationOptions
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalComposeUiApi::class)
+@Composable
+fun GroupNameScreen(
+    newGroupState: GroupMetadataState,
+    onGroupNameChange: (TextFieldValue) -> Unit,
+    onContinuePressed: () -> Unit,
```

Comment:
Hmm, maybe we can merge `onGroupNameChange` and `onContinuePressed` to a single function and pass `TextFieldValue` on `onContinuePressed` ?